client/src/modules/fill_form/fill_form.service.js

```diff
@@ -0,0 +1,86 @@
+angular.module('bhima.services')
+  .service('FillFormService', FillFormService);
+
+FillFormService.$inject = ['PrototypeApiService'];
+
+/**
+ * @class FillFormService
+ * @extends PrototypeApiService
+ *
+ * @description
+ * Encapsulates common requests to the /fill_form/ URL.
+ */
+function FillFormService(Api) {
+  const service = new Api('/fill_form/');
+
+  service.formatData = formatData;
+  service.restoreImage = restoreImage;
+  service.formatConstraint = formatConstraint;
+
+  function formatData(form, data) {
+    const dataSurveyForm = {};
+    // Ici l'on verifie les occurences des donnees existantes
+    // S'il sont multiples afin de les affecter dans des tableaux
+    form.forEach(f => {
+      data.forEach(d => {
+        if ((f.id === d.survey_form_id) && (f.type === '4')) {
```

Comment:
![image](https://user-images.githubusercontent.com/8035588/66905516-e89edb00-effd-11e9-99f7-e39e30325761.png)
It's form multiple value

Reply:
Can you instead use the label?  It will make it much easier for a developer to read.  Like this:
```js
f.label === 'FORM.LABELS.SELECT_MULTIPLE' // now all future developers knows what this means!
```